### PR TITLE
Sysbench plugin

### PR DESCRIPTION
**Pull request by Harshith-umesh**
## Changes introduced with this PR

Adds Sysbench plugin 

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: python/sysbench/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ADD requirements.txt /app
+ADD tests /app/tests/
+ADD configs /app/configs/
+WORKDIR /app
+
+RUN pip3 install -r requirements.txt
+RUN python3.9 test_sysbench_plugin.py
+
+ENTRYPOINT ["python3.9", "/app/sysbench_plugin.py"]
+CMD ["-f", "configs/sysbench_cpu_example.yaml", "-s", "sysbenchcpu"]
```
Comment (mkarg75) [inline python/sysbench/Dockerfile:18]: CMD should be empty ideally. 
The ENTRYPOINT for stressng looks like this:
`ENTRYPOINT ["/plugin/stressng_plugin.py -f stressng_example.yaml"]`

File: python/sysbench/sysbench_plugin.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+
```
Comment (mfleader) [inline python/sysbench/sysbench_plugin.py:2]: format this file to PEP 8

File: python/sysbench/test_sysbench_plugin.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
```
Comment (mfleader) [inline python/sysbench/test_sysbench_plugin.py:2]: format this file to PEP 8

**Review by mfleader — CHANGES_REQUESTED** [review]

File: python/sysbench/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,25 @@
+FROM quay.io/centos/centos:stream8
+
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip 
+RUN curl -s https://packagecloud.io/install/repositories/akopytov/sysbench/script.rpm.sh | bash
```
Comment (dustinblack) [inline python/sysbench/Dockerfile:4]: Please update this installation method to be explicit about the version of sysbench being installed.

File: python/sysbench/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+WORKDIR /app
+
+RUN pip3 install -r requirements.txt
+RUN python3.9 test_sysbench_plugin.py
+
+ENTRYPOINT ["python3.9", "/app/sysbench_plugin.py"]
+CMD []
+
+LABEL org.opencontainers.image.title="Sysbench Arcalot Plugin"
+LABEL org.opencontainers.image.source="https://github.com/arcalot/arcaflow-plugins/python/sysbench/blob/main/sysbench_plugin.py"
```
Comment (dustinblack) [inline python/sysbench/Dockerfile:21]: This should point to to the directory of the plugin, not to the plugin python file.

**Review by dustinblack — CHANGES_REQUESTED** [review]

File: python/sysbench/sysbench_plugin.py (hunk 2 above)
Comment (Harshith-umesh) [reply python/sysbench/sysbench_plugin.py:2]: done

File: python/sysbench/test_sysbench_plugin.py (hunk 3 above)
Comment (Harshith-umesh) [reply python/sysbench/test_sysbench_plugin.py:2]: done

File: python/sysbench/Dockerfile (hunk 5 above)
Comment (Harshith-umesh) [reply python/sysbench/Dockerfile]: done

File: python/sysbench/Dockerfile (hunk 4 above)
Comment (Harshith-umesh) [reply python/sysbench/Dockerfile:4]: done

**Review by dustinblack — APPROVED** [review]
Great stuff!

**Review by mfleader — APPROVED** [review]

**Pull request merged**